MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -386,32 +387,33 @@ private static TranscodeReason GetTranscodeReasonForFailedCondition(ProfileCondi
         /// <param name="type">The <see cref="DlnaProfileType"/>.</param>
         /// <param name="playProfile">The <see cref="DirectPlayProfile"/> object to get the video stream from.</param>
         /// <returns>The normalized input container.</returns>
-        public static string? NormalizeMediaSourceFormatIntoSingleContainer(string inputContainer, DeviceProfile? profile, DlnaProfileType type, DirectPlayProfile? playProfile = null)
+        public static string NormalizeMediaSourceFormatIntoSingleContainer(string inputContainer, DeviceProfile? profile, DlnaProfileType type, DirectPlayProfile? playProfile = null)
         {
-            if (string.IsNullOrEmpty(inputContainer))
+            if (!inputContainer.Contains(',', StringComparison.OrdinalIgnoreCase) || profile is null)
             {
-                return null;
+                return inputContainer;
             }
 
-            var formats = ContainerProfile.SplitValue(inputContainer);
-
-            if (profile is not null)
+            var formats = inputContainer.SpanSplit(',');
```

Comment:
Maybe add `ContainerHelper.SpanSplit` for consistency?

Reply:
This is a generic method from our string extension.

Reply:
I was thinking of passing `RemoveEmptyEntries`.